src/components/ul/ul.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+
+const styleSheet = createStyleSheet('Ul', () => ({
```

Comment:
You don't need this since it is empty, rather accept className as a prop and send it in to className